### ISSUE TITLE
navNext property not working as expected

### ISSUE THREAD

**Issue by HartiganHM** [airbnb/react-dates#1219]
I'm using both `DatePickerSingleDateController` and `DayPickerRangeController` in my application and am having consistent issues with the `navNext` property. It works _most_ of the time with `DatePickerSingleDateController`, but never seems to work with `DayPickerRangeController`. Here's the gist of what I'm experiencing:

`react-dates v 16.6.1`
`react v 16.3.2`

**`DayPickerRangeController` Setup**

```javascript
<DayPickerRangeController
  startDate={startDate || null}
  endDate={endDate || null}
  onDatesChange={({ startDate, endDate }) =>
    this.setState({ startDate, endDate })
  }
  focusedInput={focusedInput || null}
  onFocusChange={focusedInput => this.setState({ focusedInput })}
  numberOfMonths={numberOfMonths || 1}
  navPrev={<NavButton fontSize={'1.5em'}>chevron_left</NavButton>}
  navNext={<NavButton fontSize={'1.5em'}>chevron_right</NavButton>}
  keepOpenOnDateSelect
  hideKeyboardShortcutsPanel
/>
```

The nav buttons are supposed to be blue with a white arrow. I've changed the color to red inorder to highlight the issue. After inspecting the elements, it looks like the component being passed to `navPrev` is wrapped in an extra parent component and has a number of styling classes from `react-dates`. This same behavior is not happening with `navNext` which (I assume) is causing the styling issues.

<img width="1229" alt="screen shot 2018-06-19 at 12 44 12 pm" src="https://user-images.githubusercontent.com/24458700/41617727-84efb5d2-73be-11e8-8a57-0971544bdc81.png">

**`DayPickerSingleDateController` Setup**

```javascript
<DayPickerSingleDateController
  date={date || null}
  placeholder={placeholder || null}
  onDateChange={date => this.handleDateChange(date)}
  focused={true}
  onFocusChange={({ focused }) => this.setState({ focused })}
  numberOfMonths={1}
  isDayBlocked={this.isDayBlocked}
  isOutsideRange={this.isOutsideRange}
  keepOpenOnDateSelect
  navPrev={<NavButton fontSize={'1.5em'}>chevron_left</NavButton>}
  navNext={<NavButton fontSize={'1.5em'}>chevron_right</NavButton>}
  hideKeyboardShortcutsPanel
/>
```

This shows an area where `navNext` is working with almost an identical setup for `DayPickerSingleDateController`. As you can see, both `navPrev` *and* `navNext` are being wrapped in the extra parent component from `react-dates`.

<img width="1202" alt="screen shot 2018-06-19 at 12 42 07 pm" src="https://user-images.githubusercontent.com/24458700/41617658-53dd75f6-73be-11e8-8fb1-bc0a0a9e5208.png">

Any help is _greatly_ appreciated. If you need any additional information, please reach out!

**Comment by majapw** [comment]
Hello! I'm actually working on a PR for this RIGHT NOW. :) we'll be
removing all styling from the custom nav which will hopefully solve your
issue.

**Comment by HartiganHM** [comment]
@majapw Wow! Awesome fast response! 

Glad to hear it's been flagged up already. Love what you're doing with `react-dates`, it's been a lot of fun to work with. I'll watch for the update! Any ETA?

**Comment by majapw** [comment]
Hi @HartiganHM, glad you're enjoying the package! :) This feature is available in v17.0.0. Note that you will also have to position the custom nav yourself (it really did strip out *all* styles). 

Let me know if that in itself is a struggle or has any unexpected behavior. We could probably try to improve on this (maybe have position be a prop with a default value or something?).
